src/Classes/Providers/WindowsFeatureParcelProvider.ps1

```diff
@@ -41,10 +41,24 @@ class WindowsFeatureParcelProvider : ParcelProvider
     [bool] TestPackageInstalled([ParcelPackage]$_package)
     {
         if ($this.IsOptionalFeature($_package)) {
-            return ((Get-WindowsOptionalFeature -Online -FeatureName $_package.Name -ErrorAction Ignore).State -ieq 'enabled')
+            $checkDismPackage = dism /online /get-featureinfo /featurename:$_package.Name 
+            $checkDismPackageState = $checkDismPackage -match "State"
+            if ($checkDismPackageState -like "*Disabled*")
```

Comment:
I'd say for this, and the line above, use `-imatch` and `-ilike` - in case they change the output for some random reason!

Reply:
agreed!
```suggestion
            if ($checkDismPackageState -ilike "*Disabled*")
```